odk1-src/aggregate-tomcat.rst

```diff
@@ -33,143 +34,96 @@ Database systems
+    .. group-tab:: macOS
 
-   - For PostgreSQL, download and install the appropriate binary package from `PostgreSQL download site <https://www.postgresql.org/download/>`_. Be sure to set the password for the postgres (root) user and set the default character set and collation sequence.
+      Download `the PostgreSQL 10.5 Postgres.app DMG installer <https://postgresapp.com/downloads.html>`_, and open it.
 
-   - For either database, you should ensure that the default character set is configured to be UTF-8 and that the collation sequence (dictionary order) is set appropriately for your circumstances. If it isn't, any non-Latin characters may display as question marks. Refer to the character set and collation sections of your database's documentation for how to do this.
+    .. group-tab:: Ubuntu
 
-   - For Microsoft SQL Server or Azure SQL Server, you should configure these with UTF-8 character sets and to use Windows authentication. When using Windows authentication, the user under which the webserver executes must be granted permissions to access the SQL Server instance. The install wizard for ODK Aggregate will produce a :file:`Readme.html` file that contains additional information on how to complete the configuration of the database and webserver service.
+      .. code-block:: console
 
-5. Download and install `ODK Aggregate <https://github.com/opendatakit/aggregate/releases/latest>`_. Select the latest release for your operating system.
+        sudo apt-get install postgresql-10
 
-   .. note::
+7. Install `ODK Aggregate <https://github.com/opendatakit/aggregate/releases/latest>`_. Select the latest release for your operating system.
```

Comment:
Should we also mention the Aggregate-CLI as an alternative method to install Aggregate on a server?